packages/ripple-nuxt-ui/lib/module.js

```diff
@@ -25,16 +25,19 @@ module.exports = function nuxtRipple (moduleOptions) {
       ]
     }
     // Add svg-sprite-loader to create svg sprite
-    config.module.rules.push({
-      test: /\.svg$/,
-      include: [
-        resolve(__dirname, rippleIconPath),
-        resolve(__dirname, `${appDir}/assets/ripple-icon/`)
-      ],
-      use: [
-        'svg-sprite-loader',
-        'svgo-loader'
-      ]
-    })
+    const svgLoader = config.module.rules.find((rule) => rule.use && rule.use.find(r => r === 'svgo-loader'))
```

Comment:
Should we just remove this duplicate loader from `ripple-nuxt-tide` as it already be added in https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-ui/lib/module.js#L28 ? The `ripple-nuxt-ui` is a dependency of this module.